mne/io/tests/test_meas_info.py

```diff
@@ -136,6 +135,7 @@ def test_info():
         info_str = '%s' % obj.info
         assert_equal(len(info_str.split('\n')), len(obj.info.keys()) + 2)
         assert_true(all(k in info_str for k in obj.info.keys()))
+        assert '2002-12-03 19:01:10 GMT' in repr(obj.info), repr(obj.info)
```

Comment:
If assert message is provided, shouldn't it be more descriptive? 
Otherwise, pytest reporting should be enough.

```py
import pytest
my_string = 'a aa a a aa aaaaa a a aaaa'

@pytest.mark.parametrize("substring", ['a', 'x'])
def test_msg(substring):
    assert substring in my_string, my_string

@pytest.mark.parametrize("substring", ['a', 'x'])
def test_no_msg(substring):
    assert substring in my_string
``` 

![img](http://storage8.static.itmages.com/i/17/0911/h_1505129667_3598044_a671e31632.png)

Reply:
I think the Pytest outputs are verbose enough it should be fine (a major advantage of Pytest IMO)

Reply:
So I would change this

```diff
- assert '2002-12-03 19:01:10 GMT' in repr(obj.info), repr(obj.info)
+ assert '2002-12-03 19:01:10 GMT' in repr(obj.info)
```